cppcache/integration/benchmark/RegionBM.cpp

```diff
@@ -19,6 +19,17 @@
 #include <framework/Cluster.h>
 #include <framework/Gfsh.h>
 
+// Disable warning for "extra qualifications" here.  One of the boost log
+// headers triggers this warning.  Note: use of disable pragma here is
+// intentional - attempts to use push/pop as you ordinarily should just
+// yielded a gripe from the MS tools that "warning number '4596' is not a
+// valid compiler warning". re-enabling the warning after the include
+// fails in the same way, so just leave it disabled for the rest of the
+// file.  This is safe, since the warning can only trigger inside a class
```

Comment:
There's a class defined on line 51?  Or did you mean something else by this comment?

Reply:
What I'm saying is there's _no_ class defined in this file, cause that's a thing typically done in headers.  Thus, leaving the warning disabled for the rest of the .cpp file has no effect.